tests/integration-service/README.md

```diff
@@ -0,0 +1,60 @@
+# Integration Service
+
+The Integration Service manages the lifecycle, deployment and integration testing of applications and components. It handles snapshots, environments, pipelines and deployments, ensuring applications run efficiently and are deployed correctly.
+
+
+## Happy Path Tests.
+
+Happy path testing describes tests that focus on the most common scenarios while assuming there are no exceptions or errors.
+### Basic Integration Tests:
```

Comment:
```suggestion
### Basic End-to-End Tests:
```
or
```suggestion
### Basic E2E Tests:
```
Because this isn't an Integration test, but an e2e test.